velox/functions/lib/window/RegistrationFunctions.h

```diff
@@ -34,4 +34,24 @@ void registerRowNumberBigint(const std::string& name);
 // Register the Spark function row_number() with the integer data type
 // for the return value.
 void registerRowNumberInteger(const std::string& name);
+
+// Register the Presto function rank() with the bigint data type
+// for the return value.
+void registerRankBigint(const std::string& name);
+
+// Register the Spark function rank() with the integer data type
+// for the return value.
+void registerRankInteger(const std::string& name);
+
+// Register the Presto function dense_rank() with the bigint data type
```

Comment:
There isn't a need for these comments that just state what the function is doing. The meaning is explicit from the name.

Reply:
Update.